library/src/morpheus-master/sass/modules/swapped-view/_base.scss

```
@@ -5,13 +5,18 @@
 			.#{$namespace}topHeader{
 				background: $swapped-view-primary;
 			}
+			.#{$namespace}headerLogo{
+				background: $swapped-view-primary;
```

I'm not a fan of the duplication of this colour on both the topHeader and the headerLogo. It looks like it is done for the fixed header that stays at the top when scrolling down the page. Maybe this could have been done differently. 

For now, since this is the same as the rule above and will have the same colour, I'd suggest merging these two rules together:

```suggestion
.#{$namespace}topHeader, .#{$namespace}headerLogo {
				background: $swapped-view-primary;
}
```